### ISSUE TITLE
Diverse troubles, like reinitialisation of all sets after quitting the game

### ISSUE THREAD

**Issue by BluEEberry40** [OldUnreal/UnrealTournamentPatches#1223]
Hi,

I find many issues with this v469c. I'm 100% sure the problem comes from my side, and not from the patch itself, but now I have to find the way to fix it and I won't manage by myself.

Before detailing any issue, I need to mention that my current version before installing the patch is v436: what version exactly does the patch update the game from? I feel I have missed _a few_ episodes...

So first of all, the .exe installer doesn't work for me: I choose the correct path to install the game (the one by default is not the right one), then launch the installation, but the installation gauge remains stuck, it just won't upload at all.

So I have to opt for the manual installation. When it's done, the game is launched and, as expected and that's normal, I have to choose the right video configuration to launch the game (XOpenGL, Direct 3D Support, etc...), and also the mouse configuration (that's new to me...). Note that the XOpenGL still doesn't work, it already didn't work before. So I choose the Direct 3D Support, then launch the game, everything seems fine after I put the right game resolution and turned into 32 bits... But now, every time I quit the game, and launch the game again, all the parameters have reinitialised. I have to re-choose the video and mouse configuration, launch the game, reset the resolution and 32 bits, and if I edited my character, the bots, the mutators, etc..., everything has been reinitialised as well, and there is no way to break this cycle.

Also, the patch added a new game mode... which appears to be empty, there is no map to play on with. That's the kind of detail that makes me almost certain that my game might be wayyy too outdated, and then something expectably goes wrong when installing the patch. As far as I remember, I have never needed to install any patch ever since I have played the game. If there were updates out there in the wild, I didn't even know that there were, and I have never had any problem to run the game and play mods, custom maps, using custom models, etc...

One last thing: my game is originally in French, and now the patch turns many things into English (like the mutators already present in the original game) ... but not everything, like the new game mode is written in French. That's really weird.

What do you think? Thanks in advance :)

**Comment by SeriousBuggie** [comment]
D3D? Use D3D9, not D3D. Try show all drivers, not only certified. 

Config not saved? Ini files write-protected? No space on disk? Possible weird characters in it, so it not able read it after write. Possible need attach it here.

French? What about switch to English? All issues remains? It can be done in ini file.

**Comment by BluEEberry40** [comment]
I switched to D3D9, didn't fix anything though.

I managed to turn the game into French, thanks for the instructions. I didn't think it was that easy, or rather, I thought it would turn only the interface into English but the voices would remain in French. Anyway.

The Ini files don't seem to be write-protected, and there is enough space on disk.
I might have found something though: until now, the configs were saved here (like for many other games by the way):
This PC/Windows (C:)/Users/[Username]/AppData/Local/VirtualStore/Program Files (x86)/

But this time, playing the game doesn't update or modify anything in this directory anymore, nor even recreates the files if I have deleted them.

**Comment by NeonKnightOA** [comment]
You originally got the game in French but now you want it entirely in English? Or do you want french texts and french voices?

The "new game mode" isn't really new (more like a cut mode), like Vehicle CTF in UT2004 it just needs maps. There are some here: https://unrealarchive.org/maps/unreal-tournament/darkmatch/index.html

**Comment by BluEEberry40** [comment]
I originally got the game in French, yeah, because I'm French. I'm fluent in English, but I kept the game in French because I thought I would have had to find the game again just to get it in English, I was too lazy to do that. I think it was high time I switched to English. First I downloaded some custom voices, which are in English of course, and that sounds weird when the original voices are in French. And now the interface with the update uses a weird mix of English and broken French if I don't turn it completely to English, whereas it used to be entirely in well-translated French before the update. And there are even some display bugs in French now (see the screenshot, and note that I cannot make the settings window bigger for some reason, I think it has never been possible for me actually).

![scr_3](https://user-images.githubusercontent.com/128981995/227943118-c606d358-2f49-494b-8580-6057c6096f1e.png)


So switching to English is the best thing to do. I am used to the French voices, but I will get through it =P

Okay, things are more clear about this game mode now, thanks.

**Comment by NeonKnightOA** [comment]
You didn't need to download anything, default English voices are already included in the game regardless of which version you had. All you needed to do was to change a small line in UnrealTournament.ini: `language=frt` to `language=int`, and that's it!

**Comment by BluEEberry40** [comment]
Yeah, I had figured out already. It turned all the voices to English, and no more language mix in the interface either.

Anyway, the main problem is still not fixed though: always reinitialising everything every time I reopen the game :(

**Comment by BluEEberry40** [comment]
Hi,

Problem fixed, all I had to do was to install the game outside the C:/Program Files (x86)/ directory. I actually installed it in a folder on the Desktop, because apparently the permissions actually were the problem here... even though it's not any of a problem as long as I don't install the patch. But at least now I know I can play once again with the updated game, and not suffer any problem in the future. And once again, none of the files were write-protected, I guess it was just about admin settings or something (I can edit stuff in the Program Files (x86) alright, but I have to confirm every of my actions every time I do, but otherwise that's not really a problem... except here).

I used to install it in C:/Program Files (x86)/GT Interactive/ to be more precise, because that was the directory of installation by default of the original game. That's actually the first time I have to install a game somewhere else. And now, every time I play the game, the .ini files needed for all the settings edit themselves directly in the game folder, not this folder anymore:
This PC/Windows (C:)/Users/[Username]/AppData/Local/VirtualStore/Program Files (x86)/

I'm not a coder, I have no experience whatsoever, how all this works and especially why it works that way is totally beyond me.

**Comment by stijn-volckaert** [comment]
This is a permission issue. We don't have a fix for this yet, so I'll leave this ticket open for now

**Comment by Chamberly** [comment]
I recently re-installed a new batch of v436 files straight from my CD 1 & 2, then downloaded the v469d patch. I too also had the issue of file permission but that's because likely I did not install it as an admin(?). But I went through all of my folders, uncheck the read-only for each folders and also added the WRITE permission to my user profile for each folders. No issue after that.
Windows OS.
